README.md

```diff
@@ -164,12 +165,17 @@ runtimeOnly "org.postgresql:postgresql:42.2.18"
 
 You may use the following properties (typically in application.yml) to configure the Camunda BPM integration.
 
-| Prefix               |Property          | Default                                      | Description            |
-|----------------------|------------------|----------------------------------------------|------------------------|
-| camunda.bpm          | .history-level   | auto                                         | Camunda history level, use one of [`full`, `audit`, `activity`, `none`, `auto`]. `auto` uses the level already present in the database, defaulting to `full`. |
-| camunda.bpm.database | .schema-update   | true                                         | If automatic schema update should be applied, use one of [`true`, `false`, `create`, `create-drop`, `drop-create`] |
-| camunda.bpm.telemetry| .telemetryReporterActivate | true                               | Enable to report anonymized data about the installation to Camunda |
-| camunda.bpm.telemetry| .initializeTelemetry       | false                              | Enable to report anonymized data about the process engine usage to Camunda |
+| Prefix                |Property          | Default                                      | Description            |
+|-----------------------|------------------|----------------------------------------------|------------------------|
+| camunda.bpm           | .history-level   | auto                                         | Camunda history level, use one of [`full`, `audit`, `activity`, `none`, `auto`]. `auto` uses the level already present in the database, defaulting to `full`. |
+| camunda.bpm.database  | .schema-update   | true                                         | If automatic schema update should be applied, use one of [`true`, `false`, `create`, `create-drop`, `drop-create`] |
+| camunda.bpm.telemetry | .telemetryReporterActivate | true                               | Enable to report anonymized data about the installation to Camunda |
+| camunda.bpm.telemetry | .initializeTelemetry       | false                              | Enable to report anonymized data about the process engine usage to Camunda |
+| camunda.bpm.admin-user| .id             |                                               | If present, the Camunda admin account will be created by this id  |
```

Comment:
* "the Camunda admin account" -> "a Camunda admin account"
* add a hint that a group and authorizations are also created.